docs/reference/map.rst

```diff
@@ -5,61 +5,14 @@ Maps (`sunpy.map`)
 
 .. module:: sunpy.map
 
-.. testsetup::
-
-    import sunpy
-
-.. currentmodule:: sunpy.map
-
-Overview
-========
-One of core classes in sunpy is a Map. A sunpy Map object is simply a
-spatially-aware data array, often an image. In order to make it easy to work
-with image data in sunpy, the Map object provides a number of methods for
-commonly performed operations.
-
-2D map objects are subclasses of `~sunpy.map.GenericMap` and these objects are
-created using the Map factory `~sunpy.map.Map`.
-
-A number of instrument are supported by subclassing this base object. See
-:ref:`map-sources` to see a list of all of them. More complex subclasses are also
-available. See :ref:`map-classes`.
-
-.. todo :
-    1. Map factory and registration
-    2. MapBase and Generic Map
-    3. MapMeta and the separation from the file io
-
-
-Creating Map Objects
-====================
-sunpy Map objects are constructed using the special factory
-class `~sunpy.map.Map`: ::
-
-    >>> x = sunpy.map.Map('file.fits')  # doctest: +SKIP
-
-The result of a call to `~sunpy.map.Map` will be either a `~sunpy.map.mapbase.GenericMap` object,
-or a subclass of `~sunpy.map.mapbase.GenericMap` which either deals with a specific type of data,
-e.g. `~sunpy.map.sources.sdo.AIAMap` or `~sunpy.map.sources.soho.LASCOMap`
-(see :ref:`map-classes` to see a list of all of them), or if no
-instrument matches, a 2D map `~sunpy.map.mapbase.GenericMap`.
-
```

Comment:
Is this information captured somewhere else?

Reply:
Err, I do not think it is but for me I didn't think it adds much but I can restore.

Reply:
I added it back.